edx_proctoring/api.py

```diff
@@ -1497,14 +1497,11 @@ def _get_timed_exam_view(exam, context, exam_id, user_id, course_id):
         progress_page_url = ''
         try:
             progress_page_url = reverse(
-                'courseware.views.progress',
+                'courseware.views.views.progress',
                 args=[course_id]
             )
         except NoReverseMatch:
```

Comment:
The exception block is for running this in an environment where that view isn't defined (like unit tests)